README.md

```diff
@@ -27,11 +26,30 @@ in the PPM working group at the IETF. This support is still experimental, and is
 evolving along with the DAP and VDAF specifications. Formal security analysis is
 also forthcoming. Prio3 should not yet be used in production applications.
 
+### Draft versions and release branches
+
+The `main` branch is under continuous development and will usually be partway between VDAF drafts.
+libprio uses stable release branches to maintain implementations of different VDAF draft versions.
+Crate `prio` version `x.y.z` is released from a corresponding `release/x.y` branch. We try to
+maintain [SemVer][semver] compatibility, meaning that API breaks only happen on minor version
+increases (e.g., 0.10 to 0.11.
+
+| Git branch | Draft version | Conforms to specification? | Status |
+| ---------- | ------------- | --------------------- | ------ |
+| `release/0.8` | [`draft-irtf-cfrg-vdaf-01`][vdaf-01] | Yes | Supported |
+| `release/0.9` | [`draft-irtf-cfrg-vdaf-03`][vdaf-03] | Yes | Unmaintained as of September 22, 2022 |
+| `release/0.10` | [`draft-irtf-cfrg-vdaf-03`][vdaf-03] | Yes | Supported |
+| `main` | `draft-ietf-ppm-dap-04` ([forthcoming][vdaf-04]) | Partially | Supported, unstable |
```

Comment:
We're not yet on draft 04, and as far as I know have not implemented any of its features. In fact, I wonder if we want to keep open the possibility of cutting 0.11 with 03 and wait for a future version for 04.

Reply:
That makes sense -- we definitely want to get `SumVec` out before VDAF-04 arrives.